workspace/src/control/include/control/control.h

```diff
@@ -0,0 +1,162 @@
+/******************************************************************************
+ * BSD 3-Clause License
+ *
+ * Copyright (c) 2022 University of Wisconsin - Madison
+ *
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice,
+ *    this list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * 3. Neither the name of the copyright holder nor the names of its
+ *    contributors may be used to endorse or promote products derived from
+ *    this software without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ ******************************************************************************/
+
+#ifndef CONTROL_H
+#define CONTROL_H
+
+#include "rclcpp/rclcpp.hpp"
+#include "nav_msgs/msg/path.hpp"
+#include "art_msgs/msg/vehicle_input.hpp"
+#include "art_msgs/msg/vehicle_state.hpp"
+#include "geometry_msgs/msg/pose_stamped.hpp"
+
+/**
+ * @brief Control Node class
+ *
+ * Publishes steering, throttle & breaking values
+ * Subscribes to Navigation Path & ART/Chrono Vehicle State messages
+ */
+class control : public rclcpp::Node {
+    /*******************************
+                PRIVATE
```

Comment:
need to add `private:` if you want these variables to be private. Also, some variables end in `_` and some don't. Maybe wanna be consistent? Also, Chrono uses `m_<private variable>`. I personally think we should keep this standard, but up for discussion.

Reply:
Isn't the default access specifier in C++ classes `private` ? I've anyways added `private:` for good measures.

Reply:
Ya you're write, idk what I was thinking. Being explicit about it is good though, so agree with your change